test/PolyhedralGeometry/timing.jl

```diff
@@ -4,7 +4,22 @@
     # to avoid unnecessary failures we skip the timing tests
     haskey(ENV, "JULIA_PKGEVAL") && return
 
-    using Oscar
+    # If running multiple workers, and many OMP threads, CPU oversubscription may occur
+    # which fails the timing tests. In this situation, skip instead.
+    try
+        if (length(workers())>1)
+            if haskey(ENV, "OMP_NUM_THREADS")
+                if (ENV["OMP_NUM_THREADS"] != "1")
+                    return
+                end
+            else
+                return
+            end
+        end
```

Comment:
```suggestion
        if (length(workers()) > 1)
            haskey(ENV, "OMP_NUM_THREADS") || return
            ENV["OMP_NUM_THREADS"] == "1" || return
        end
```
This would be easier to read and more idiomatic.